implement ability to use multiple webhook targets

JohnyTheCarrot opened the pull request:
Per Bursting's request, the ability to use multiple webhooks defined in the environment variables.

JohnyTheCarrot committed:
implement ability to use multiple webhook targets

roeeyud commented on src/index.ts:
```diff
@@ -10,21 +10,35 @@ const port: number = parseInt(process.env.PORT ?? '3000');
-  return fetch(process.env.WEBHOOK!, {
-    method: 'POST',
+function newIssue(payload: IncomingLinearWebhookPayload, webhookTarget: string) {
+  const target = process.env[`WEBHOOK_${webhookTarget}`];
```
maybe call toUppercase.

JohnyTheCarrot committed:
call toUppercase just in case

While env variables don't seem to be case sensitive on Windows, it might be case sensitive on other operating systems.
